src/codec/MapClearCodec.ts

```diff
@@ -0,0 +1,41 @@
+/* tslint:disable */
+import ClientMessage = require('../ClientMessage');
+import {BitsUtil} from '../BitsUtil';
+import Address = require('../Address');
+import {Data} from '../serialization/Data';
+import {MapMessageType} from './MapMessageType';
+
+var REQUEST_TYPE = MapMessageType.MAP_CLEAR;
+var RESPONSE_TYPE = 100;
+var RETRYABLE = false;
+
+
+export class MapClearCodec {
+
+
+    static calculateSize(name:string) {
+        // Calculates the request payload size
+        var dataSize:number = 0;
+        dataSize += BitsUtil.calculateSizeString(name);
+        return dataSize;
+    }
+
+    static encodeRequest(name:string) {
+        // Encode request into clientMessage
+        var clientMessage = ClientMessage.newClientMessage(this.calculateSize(name));
+        clientMessage.setMessageType(REQUEST_TYPE);
+        clientMessage.setRetryable(RETRYABLE);
+        clientMessage.appendString(name);
+        clientMessage.updateFrameLength();
+        return clientMessage;
+    }
+
+    static decodeResponse(clientMessage:ClientMessage, toObjectFunction:(data:Data) => any) {
+        // Decode response from client message
```

Comment:
There are some decodeResponse functions that should be removed like this one.
We still put a comment for it to show that it is there but removed as no one will call it.
```
//static decodeResponse(clientMessage:ClientMessage, toObjectFunction:(data:Data) => any)
```